packages/engine/Cargo.toml

```diff
@@ -43,6 +43,8 @@ tracing-appender = "0.2.0"
 tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.7", features = ["env-filter", "json"] }
 tracing-texray = { version = "0.1.2", optional = true }
+# Only temporary, use the crates.io release when ArrayBuffer::new_backing_store_from_ptr is available
+v8 = { git = "https://github.com/hashdeps/rusty_v8-1", branch = "backing-store-from-ptr" }
```

Comment:
Commenting here so we can reply inline.

> # 📜 Does this require a change to the docs?
> No

This is incorrect. It does. Can you please read through the README and look at how the build instructions have changed. For example we do not need to ask users to download V8 anymore, that should be clear.


Furthermore the rusty_v8 docs mention this:

> Building V8 takes over 30 minutes, this is too slow for me to use this crate. What should I do?
> Install [sccache](https://github.com/mozilla/sccache) or [ccache](https://ccache.dev/). Our build scripts will detect and use them. Set the $SCCACHE or $CCACHE environmental variable if it's not in your path.

I haven't seen any mention of that or consideration for it in this PR. Can we please investigate (probably should include looking through the issues on the rusty_v8 repo) when people use this, and if we should recommend it in the README.

Reply:
Oh I didn't think of the README.

There are definitely things to change.
Do we want to document how to build v8 from source in this PR or the next one when we'll compile v8 from source ourselves?

Reply:
I wasn't sure about the ARM-based Macs so I left it there, do you think it's still necessary?
https://github.com/hashintel/hash/pull/414/commits/86c818d8c3bf2cc7e14ae039f135150152c6c79e

Reply:
Added to [Asana task](https://app.asana.com/0/1201893074552404/1201907929549874/f) (_internal_)